ospfd/ospf_spf.c

```diff
@@ -905,7 +905,9 @@ static unsigned int ospf_nexthop_calculation(struct ospf_area *area,
 				 * somehow.
 				 */
 				if (area->ospf->ti_lfa_enabled
-				    || (oi && oi->type == OSPF_IFTYPE_POINTOPOINT)) {
+				    || (oi && oi->type == OSPF_IFTYPE_POINTOPOINT)
+				    || (oi && oi->type == OSPF_IFTYPE_POINTOMULTIPOINT
+					   && oi->address->prefixlen == IPV4_MAX_BITLEN)) {
```

Comment:
@qlyoung do you know a  good fix for this warning? 

see https://github.com/FRRouting/frr/pull/8153#issuecomment-811573353

Reply:
@polychaeta said nothing after the fix, I'm not sure what warning you're referring to

Reply:
@qlyoung , thanks. Just wanted to make sure that If I merge this PR the warning wont trickle down to allow new PRs.